### ISSUE TITLE
Sums of split transactions on reconcilliation screen

### ISSUE THREAD

**Issue by andyshilton** [firefly-iii/firefly-iii#4046]
**Description**

I wondered if it would be possible to add a sum of split transations when reconciling, just like there is on the account display. I split my entries so I can categorise parts into what I've spent it on. eg, one Sainsbury's transaction may contain some for food, some for cleaning products etc and I love that I can split it in FFiii to know what I've spent where.
However, when reconciling I've found myself adding up the parts again to check they match my statement. So I wondered if it would be possible to add this into FFiii?

Entering a split transaction:
![image](https://user-images.githubusercontent.com/16556349/98466545-17b75a00-21c8-11eb-9438-3a22ebc3b5ab.png)

Account display where it shows the total:
![image](https://user-images.githubusercontent.com/16556349/98466556-2b62c080-21c8-11eb-9a50-22ea64d4d961.png)

Reconiliation page where no sum is present:
![image](https://user-images.githubusercontent.com/16556349/98466598-7250b600-21c8-11eb-8268-171724008871.png)

**Solution**

Add a sum above the split lines on the reconciliation page, just like it does on the account page.
**What are alternatives?**

Other than using a calculator, I'm not sure of an alternative.

**Comment by JC5** [comment]
I'll write it down for the long term. Current view/layout, it's not going to change I'm afraid.

**Comment by andyshilton** [comment]
Hi @JC5  I didn't really mean to change the design/layout. I just wondered if, with the existing design and layout, it would be possible to display a sum of split transactions so that you can see the total for comparing to your statement. Would that be something that's possible, without changing the design?

**Comment by viharm** [comment]
Yes, I too find myself adding up the split transactions to check if they reconcile with the single _real_ transaction in the bank statement.

I have the same request. The act of reconciliation is to compare what's in Firefly vs the bank's statement. And the latter does not split transactions. If I had the knowledge, I'd love to contribute :-(
